Cleanups to JPetLogger

grey277 opened the pull request:
Display thread id only when multi threading is enabled, reorder displayed information

wkrzemien commented on Core/JPetLogger/JPetLogger.cpp:
```diff
@@ -14,32 +14,43 @@
+  out_stream << "[" << boost::filesystem::path(fullpath.get()).filename().string() << ":";
+  out_stream << fullfunction << "@";
+  out_stream << boost::log::extract<int>("Line", rec) << "]";
+  if (JPetManager::getManager().areThreadsEnabled())
```
Couldn't you just use some argument as bool variable, passed to this function?
I don't like too much adding this dependency to JPetManager
I would prefer have formatter with an additional bool argument,

grey277 commented:
I removed JPetManager dependency. Beside of that I refactored JPetLogger to reduce usage of getInstance() and hide it in static function.

Example log message:
1: <error> the option '--file' is required but missing [JPetManager.cpp:parseCmdLine@94]
2: <error> While parsing command line arguments [JPetManager.cpp:run@45]
3: <info> ======== Starting processing all tasks: 17.02.2019 20:11 ========
 [JPetManager.cpp:run@54]
4: <error> No local database file found. [JPetParamManager.cpp:generateParamManager@37]

JPETTomographyTester commented on Core/JPetManager/JPetManager.cpp:
```diff
@@ -98,33 +92,29 @@ std::pair<bool, std::map<std::string, boost::any> >  JPetManager::parseCmdLine(i
 
-void JPetManager::useTask(const std::string& name, const std::string& inputFileType, const std::string& outputFileType, int numTimes)
-{
+void JPetManager::useTask(const std::string& name, const std::string& inputFileType, const std::string& outputFileType, int numTimes) {
```
**Reporter**: CPPCHECK
**Rule**: unusedFunction
**Severity**: INFO
**File**: /var/jenkins_home/workspace/j-pet-framework-PRs-root-5/Core/JPetManager/JPetManager.cpp L100

The function &#39;useTask&#39; is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a801314417>*
